src/StatusQ/Components/StatusChatInfoToolBar.qml

```diff
@@ -104,5 +102,4 @@ Item {
             }
         }
     }
-
```

Comment:
Surprised these would affect the alignment of the chatlist!

Reply:
since ever!